src/dmd/frontend.d

```diff
@@ -146,6 +149,9 @@ void initDMD(
     Objc._init();
     builtin_init();
     FileCache._init();
+
+    version (CRuntime_Microsoft)
+        initFPU();
```

Comment:
@rainers is there a need to restore this to whatever it was before?

Reply:
I don't think so. It changes rounding control of the FPU to higher precision, so it should only make a difference for code that needs the exact default precision. That's unlikely to appear at the same time as this library.

Reply:
Is it restored automatically when the application exists?

Reply:
> Is it restored automatically when the application exists?

The FPU state exists per thread, so no other processes affected.